../gopkg.in/sourcemap.v2/mappings.go:8:2: use of internal package not allowed

Hi,

I get an error (`../gopkg.in/sourcemap.v2/mappings.go:8:2: use of internal package not allowed`) when I try to run the following code:

```go
package main

import (
	"github.com/robertkrimen/otto"
)

func main()  {
	vm := otto.New()

	vm.Run(`console.log(1 + 1)`)
}
```

I am not really sure how to debug it.

--- a/file/file.go
+++ b/file/file.go
@@ -6,7 +6,7 @@ import (
 	"fmt"
 	"strings"
 
-	"gopkg.in/sourcemap.v2"
+	"gopkg.in/sourcemap.v1"
 )
 
 // Idx is a compact encoding of a source position within a file set.

--- a/parser/parser.go
+++ b/parser/parser.go
@@ -40,11 +40,10 @@ import (
 	"io"
 	"io/ioutil"
 
-	"gopkg.in/sourcemap.v2"
-
 	"github.com/robertkrimen/otto/ast"
 	"github.com/robertkrimen/otto/file"
 	"github.com/robertkrimen/otto/token"
+	"gopkg.in/sourcemap.v1"
 )
 
 // A Mode value is a set of flags (or 0). They control optional parser functionality.

